qiskit/providers/ibmq/experiment/experimentservice.py

```diff
@@ -110,6 +111,7 @@ def experiments(
         """Retrieve all experiments, with optional filtering.
 
         Args:
+            limit: Number of experiments to retrieve. ``None`` indicates no limit.
```

Comment:
>``None`` indicates no limit.

This is half true, there is a limit enforced on the server side regardless. This is really saying there is no limit imposed from the client side.

Reply:
Yes. If the user sets `limit=None`, then qiskit will continue querying until no marker is returned.